deploy/crds/crd-certificates.yaml

```diff
@@ -129,11 +129,13 @@ spec:
                       type: object
                       required:
                         - create
-                        - passwordSecretRef
```

Comment:
I think we now need a webhook validation that either `passwordSecretRef` or `password` field is set. Never both.

Reply:
It would indeed be cleaner, I'll add to internal/apis/config/webhook/validation/validation.go